src/card.css.ts

```diff
@@ -387,20 +387,19 @@ export const vars = {aspectRatio};
 
 export const cardContainer = style({
     position: 'relative',
-    aspectRatio,
-    '@supports': {
-        'not (aspect-ratio: 1 / 1)': {
-            display: 'flex',
-            ':before': {
-                float: 'left',
-                content: '""',
-                paddingTop: `calc(100% / ${aspectRatio})`,
-            },
-            ':after': {
-                display: 'block',
-                content: '""',
-                clear: 'both',
-            },
-        },
-    },
+    ...(aspectRatio
+        ? {
+              display: 'flex',
+              ':before': {
+                  float: 'left',
+                  content: '""',
+                  paddingTop: `calc(100% / ${aspectRatio})`,
+              },
+              ':after': {
+                  display: 'block',
+                  content: '""',
+                  clear: 'both',
+              },
+          }
+        : {}),
```

Comment:
This ternary doesn't make sense, `aspectRatio` will be always truthy
```suggestion
          display: 'flex',
          ':before': {
              float: 'left',
              content: '""',
              paddingTop: `calc(100% / ${aspectRatio})`,
          },
          ':after': {
              display: 'block',
              content: '""',
              clear: 'both',
          },
```

Reply:
fixed